### PR TITLE
Make sure we ignore whitespace in java options

### PR DESCRIPTION
**Pull request by tgodzik**
Fixes https://github.com/scalameta/metals-languageclient/issues/59

### CODE REVIEW

**Comment by tgodzik** [comment]
> Thanks for catching and fixing this! Can you also add a test case for this?

Added, does the test case make sense?

File: src/__tests__/getJavaOptions.test.ts [hunk 1]
```diff
@@ -24,6 +24,15 @@ describe("getJavaOptions", () => {
     expect(options).toEqual(proxyOptions);
   });
 
+  const malformedOptions = ["-XX:+UseNUMA   ", "-XX:+UseZGC  "];
+  const expectedOptions = ["-XX:+UseNUMA", "-XX:+UseZGC"];
```
Comment (gabro) [inline src/__tests__/getJavaOptions.test.ts]: minor, but since these two const are only used by a single test, can you move them inside?

Comment (tgodzik) [reply src/__tests__/getJavaOptions.test.ts]: Done!

**Comment by gabro** [comment]
Thanks! Feel free to cut a release (or I can take care of it tomorrow)

**Pull request merged**